src/neofs_testlib/env/env.py

```diff
@@ -198,7 +201,15 @@ def load(cls, persisted_path: str) -> "NeoFSEnv":
     def simple(cls) -> "NeoFSEnv":
         neofs_env = NeoFSEnv()
         neofs_env.deploy_inner_ring_node()
-        neofs_env.deploy_storage_nodes(count=4)
+        neofs_env.deploy_storage_nodes(
+            count=4, 
+            attrs={
+                0: ["UN-LOCODE:RU MOW", "Price:22"],
```

Comment:
different prices are doubtful to be the default setup

Reply:
These values are from the current static dev-env. And since current tests rely on this dev-env, it is considered as the default. Actually, this code can be moved to testcases dir, maybe I will do it in the future. Cause this lib should provide only the basic blocks.